app/views/pages/contact.html.erb

```diff
@@ -14,4 +14,88 @@
       Se faisant, formes, espaces et couleurs sont mis en relation les uns avec les autres privilégiant ainsi, l'harmonie de la composition dans un continuum spatio-temporel, d'une sensualité toute organique.
     </p>
   </div>
+
+  <div class="contact-wrapper">
+ <!-- ------------ Personnal Collection ------------ -->
+    <div class="exhibitions-box" id="personnal-box">
+      <h2>Expositions Personnelles</h2>
```

Comment:
Be carefull not to forget about english translation. This one should be a line in the language locales. Same for the bio text.

Reply:
I will do it it when the design of the contact page will be done :). for more productivity. I have to clean .YML locale file too